riscv/insns/c_jalr.h

```diff
@@ -3,3 +3,10 @@ require(insn.rvc_rs1() != 0);
 reg_t tmp = npc;
 set_pc(RVC_RS1 & ~reg_t(1));
 WRITE_REG(X_RA, tmp);
+
+if (p->extension_enabled(EXT_ZICFILP)) {
+  if (ZICFILP_READ_XLPE(STATE.v, STATE.prv)) {
+    STATE.elp = ZICFILP_EXPECT_LP(insn.rvc_rs1());
+  }
+}
+
```

Comment:
Delete extraneous newline

Reply:
could name ZICFILP_READ_XLPE to ZICFILP_xLPE to match spec.

Reply:
Fixed